README.md

```diff
@@ -143,6 +176,9 @@ The `UniqueHandle` helper class and all the unique handle types and functions re
 > If `VKFW_NO_INCLUDE_VULKAN_HPP` isn't defined, then `vkfw` will use the smart handles from Vulkan-HPP directly. In the event that `VULKAN_HPP_NO_SMART_HANDLES` is defined, then `VKFW_NO_SMART_HANDLE` will also be defined if it hasn't been defined already and a compiler warning will be issued.
 > You can avoid this behavior by defining either `VKFW_NO_SMART_HANDLE` or `VKFW_NO_INCLUDE_VULKAN_HPP` before including `<vkfw/vkfw.hpp>`
 
+### `VKFW_NO_RAII_HANDLE`
+The RAII wrappers and the `raii` namespace (if `VKFW_RAII_NAMESPACE` isn't defined) will be disabled.
```

Comment:
Please add a note about this having no effect if all the unique handles are disabled.